cmd/ef_tests/types.rs

```diff
@@ -15,7 +15,7 @@ pub struct TestUnit {
     pub blocks: Vec<Block>,
     pub genesis_block_header: Header,
     #[serde(rename = "genesisRLP")]
-    pub genesis_rlp: serde_json::Value,
+    pub genesis_rlp: String,
```

Comment:
you could deseralize with serde right? Something like `#[serde(with = "ethereum_rust_core::serde_utils::bytes")]`.

Reply:
Done!